website/src/pages/documents/document.page.tsx

```diff
@@ -347,7 +355,13 @@ export const DocumentTitleHeader = (p: {
           audioUrl={p.doc.audioRecording.resourceUrl}
           showProgress
         />
+        {p.doc.audioRecording && (
+        <div>
+        {!isMobile ? <a href={p.doc.audioRecording?.resourceUrl} download={"audio"}><Button>Download Audio</Button></a> : null}
```

Comment:
Also the `download` attribute should contain the name of the document containing the audio recording, not just "audio". I think you have access to this field in the `doc` prop.